_posts/2019-01-03-elixir-kaffe-codealong.md

```diff
@@ -0,0 +1,204 @@
+# Connecting Elixir to Kafka with Kaffe: A codealong
+_The most basic example that exists_
+
+If we want to use the popular messaging system Kafka with our Elixir projects, we have a few wrappers we can choose from. One of them, [Kaffe](https://github.com/spreedly/kaffe), proved difficult to troubleshoot. Just try googling "Elixir Kaffe" yourself.
+
+In this codealong we'll build the barest of simple Elixir applications and use Kaffe to connect it to a locally running Kafka server. Later we'll cover a couple of variations to connect a dockerized Kafka server or an umbrella elixir app.
+
+This post assumes basic knowledge of Elixir and no knowledge of Kafka or Kaffe. Here is the repo with the full project: [Elixir Kaffe Codealong](https://github.com/meryldakin/elixir_kaffe_codealong).
```

Comment:
Could we transfer this repo to our org so we can keep it updated along with the lesson?

Reply:
definitely!

Reply:
You can do that with this guide https://help.github.com/articles/transferring-a-repository/#transferring-a-repository-owned-by-your-personal-account

Reply:
Looks like I don't have permission to create repos on elixirschool when I go to transfer it. @SophieDeBenedetto could you grant me that temporarily or is there a workaround?

Reply:
ah right. Its cool I just forked it. Let's use this link https://github.com/elixirschool/elixir_kaffe_codealong